tests/gis_tests/geos_tests/test_io.py

```diff
@@ -184,7 +184,7 @@ def test_empty_polygon_wkb(self):
             wkb_w.byteorder = byteorder
             for srid, hex in enumerate(hexes):
                 wkb_w.srid = srid
-                self.assertEqual(wkb_w.write_hex(p), hex)
+                self.assertEqual(wkb_w.write_hex(p), hex)  # This line fails with b'0000000000' != b'000000000300000000'
```

Comment:
@sir-sigurd, can you investigate or suggest what might be the issue?

Reply:
This looks like the change https://trac.osgeo.org/geos/ticket/680 (authored by Sergey himself). The change went in GEOS 3.6.1. I guess the test's expectations should be conditional on `geos_version_tuple()`.

Reply:
I can't check it properly as I'm on vacation, but I guess conditionals should be added to WKBWriter class. See https://github.com/django/django/commit/b90d72facf1e4294df1c2e6b51b26f6879bf2992#diff-181a3ea304dfaf57f1e1d680b32d2b76R226

Reply:
Thanks for the pointer! Done in https://github.com/django/django/pull/10014.